lib/index.js

```diff
@@ -2,8 +2,13 @@
  * Module dependencies.
  */
 
+// Replaces `var zmq = require('./bindings')`;
+var binary = require('node-pre-gyp');
+var path = require('path');
+var bindingPath = binary.find(path.resolve(path.join(__dirname, '../package.json')));
```

Comment:
It might be cleaner to `join(__dirname, '..', 'package.json')` to avoid bias towards forward slashes.